test/docker_tests/test_ldap_ui.py

```
@@ -0,0 +1,247 @@
+    project_name = f"netalertx-ldap-{int(time.time())}"
+
+    workspace_dir = pathlib.Path("/workspaces/NetAlertX")
+    if workspace_dir.exists():
+        base_dir = workspace_dir / "test" / "tmp_ldap_ui"
+        base_dir.mkdir(parents=True, exist_ok=True)
+    else:
+        base_dir = tmp_path
+
+    # Create test data directories for NetAlertX
+    _create_test_data_dirs(base_dir)
+    subprocess.run(["chmod", "-R", "777", str(base_dir / "test_data")])
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)